### PR TITLE
Add a library and settings screen to the sample

### PR DESCRIPTION
**Pull request by yschimke**
#### WHAT

image

image

#### WHY

More interaction paths, and need settings to toggle podcast buttons.

#### Checklist :clipboard:
- [ ] Add explicit visibility modifier and explicit return types for public declarations
- [ ] Run spotless check
- [ ] Run tests
- [ ] Update metalava's signature text files

### CODE REVIEW

File: media-sample/src/main/java/com/google/android/horologist/mediasample/ui/components/MediaArtwork.kt [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import coil.compose.rememberAsyncImagePainter
+import com.google.android.horologist.media.ui.state.model.MediaItemUiModel
+
+@Composable
+fun MediaArtwork(
+    modifier: Modifier = Modifier,
+    mediaItem: MediaItemUiModel
```
Comment (luizgrp) [inline media-sample/src/main/java/com/google/android/horologist/mediasample/ui/components/MediaArtwork.kt:29]: nit: required items should be before `modifier` param - same in MediaChip, UampLibraryScreen, UampSettingsScreen

File: media-sample/src/main/java/com/google/android/horologist/mediasample/ui/components/MediaArtwork.kt [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+) {
+    Image(
+        modifier = modifier,
+        painter = rememberAsyncImagePainter(mediaItem.artworkUri),
+        contentDescription = mediaItem.title ?: "Unknown",
```
Comment (luizgrp) [inline media-sample/src/main/java/com/google/android/horologist/mediasample/ui/components/MediaArtwork.kt:34]: nit: string should come from the resources file - same in MediaChip, UampLibraryScreen, UampSettingsScreen

File: media-sample/src/main/java/com/google/android/horologist/mediasample/ui/settings/SettingsScreenViewModel.kt [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+package com.google.android.horologist.mediasample.ui.settings
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewmodel.initializer
+import androidx.lifecycle.viewmodel.viewModelFactory
+import kotlinx.coroutines.flow.MutableStateFlow
+
+class SettingsScreenViewModel : ViewModel() {
+    val uiState = MutableStateFlow(UiState())
```
Comment (luizgrp) [inline media-sample/src/main/java/com/google/android/horologist/mediasample/ui/settings/SettingsScreenViewModel.kt:25]: nit: make this private backing field and expose a public StateFlow

File: media-sample/src/main/java/com/google/android/horologist/mediasample/ui/settings/SettingsScreenViewModel.kt [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    fun setPodcastControls(enabled: Boolean) {
+        uiState.value = uiState.value.copy(podcastControls = enabled)
+    }
+
+    fun setLoadItemsAtStartup(enabled: Boolean) {
+        uiState.value = uiState.value.copy(loadItemsAtStartup = enabled)
+    }
+
+    fun logout() {
+    }
```
Comment (luizgrp) [inline media-sample/src/main/java/com/google/android/horologist/mediasample/ui/settings/SettingsScreenViewModel.kt:41]: nit: add a TODO or an explicit comment saying it's meant to not do anything

**Review by luizgrp — APPROVED** [review]

File: media-sample/src/main/java/com/google/android/horologist/mediasample/ui/settings/SettingsScreenViewModel.kt (hunk 3 above)
Comment (yschimke) [reply media-sample/src/main/java/com/google/android/horologist/mediasample/ui/settings/SettingsScreenViewModel.kt:25]: Done, should go anyway, need to move the PreferencesDataStore.

**Pull request merged**